packages/performance/src/performance.ts

```diff
@@ -159,6 +159,16 @@ export class Performance {
           | TimeToFirstContentfulPaintEvent);
       });
     }
+
+    if (typeof perfMetrics !== 'undefined') {
+      perfMetrics.onFirstInputDelay(delay => {
+        this.lifecycleEvent({
```

Comment:
Is this really a lifecycle event?  It's a thing that happens once, so I think so.  But it's also the only one that concerns `duration` instead of `start`.  Should we expose a way for consumers to know which value is significant?

Reply:
This definitely does feel a bit weird but I think we are kind of stuck putting it here. Consumers at least have a way of figuring out whether it was some sort of duration event or a point in time by looking at `duration`, but I agree it's not the cleanest abstraction. IMO, this is fine for the time being.